inference-engine/tests/functional/plugin/myriad/shared_tests_instances/behavior/infer_request_dynamic.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "behavior/ov_infer_request/infer_request_dynamic.hpp"
+#include <vpu/private_plugin_config.hpp>
+#include <vpu/configuration/options/dump_internal_graph_file_name.hpp>
+#include <vpu/configuration/options/dump_all_passes_directory.hpp>
+#include <vpu/configuration/options/dump_all_passes.hpp>
+
+using namespace ov::test::behavior;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP16
+};
+
+const std::vector<std::map<std::string, std::string>> configs = {
+    {}
+};
+
+const std::vector<std::map<std::string, std::string>> HeteroConfigs = {
+    {{"TARGET_FALLBACK", CommonTestUtils::DEVICE_CPU}}
```

Comment:
Why `DEVICE_CPU` not `DEVICE_MYRIAD` here?

Reply:
because this is a device to fallback to in case one of the nodes isn't supported on MYRIAD